tests/migration/migration.go

```diff
@@ -3414,3 +3397,20 @@ func haveMigrationState(matcher gomegatypes.GomegaMatcher) gomegatypes.GomegaMat
 		}),
 	}))
 }
+
+func withKernelBoot() libvmi.Option {
+	return func(vmi *v1.VirtualMachineInstance) {
+		kernelBootFirmware := utils.GetVMIKernelBootWithRandName().Spec.Domain.Firmware
+		if vmi.Spec.Domain.Firmware == nil {
+			vmi.Spec.Domain.Firmware = kernelBootFirmware
+		} else {
+			vmi.Spec.Domain.Firmware.KernelBoot = kernelBootFirmware.KernelBoot
+		}
+	}
+}
```

Comment:
Does this trigger failure?

Reply:
Yes, the `loginToFedora` fails with kernelBoot. Investigating on it